ring-jetty-adapter/src/ring/adapter/jetty.clj

```diff
@@ -79,10 +79,16 @@
       :need (.setNeedClientAuth context-server true)
       :want (.setWantClientAuth context-server true)
       nil)
-    (if-let [exclude-ciphers (options :exclude-ciphers)]
-      (.addExcludeCipherSuites context-server (into-array String exclude-ciphers)))
-    (if-let [exclude-protocols (options :exclude-protocols)]
-      (.addExcludeProtocols context-server (into-array String exclude-protocols)))
+    (when-let [exclude-ciphers (options :exclude-ciphers)]
+      (let [ciphers (into-array String exclude-ciphers)]
+        (if (options :replace-exclude-ciphers?)
+          (.setExcludeCipherSuites context-server ciphers)
+          (.addExcludeCipherSuites context-server ciphers))))
```

Comment:
I initially tried flipping this around and having something like 
```clojure
(let [excluder (if (options :replace-exclude-ciphers?)
                         (.setExcludeCipherSuites)
                         (.addExcludeCipherSuites))]
  (excluder context-server ciphers))
```

But I a) don't know if this is idiomatic, and b) don't think this works so easily? Bit of a clojure novice.

Reply:
Unfortunately method calls don't work the same as functions. You need to wrap them in a function for this to work, but its more efficient in this case to do as you've already done.